person.rb

```diff
@@ -1,15 +1,16 @@
 require './corrector'
-
+require './rental'
 class Person
   attr_reader :id
   attr_accessor :name, :age
 
```

Comment:
- [ ] To read/write from/to @rentals object, we need to set `Getters and Setters` like so `attr_accessor :rentals`. Please kind fix that.

Reply:
- [ ] This is not implemented yet, you have to set `Getters and setters` in the `person.rb` and not `rental.rb`. Please remove that from `rental.rb` and add in the `person.rb`.